app/workers/scheduler/follow_recommendations_scheduler.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+class Scheduler::FollowRecommendationsScheduler
+  include Sidekiq::Worker
+  include Redisable
+
+  sidekiq_options retry: 0
+
+  def perform
+    # This speeds-up subsequent queries significantly
+    AccountSummary.refresh
+
+    fallback_recommendations = FollowRecommendation.safe.filtered.limit(100)
+
+    I18n.available_locales.each do |locale|
+      recommendations = []
+
+      # We can skip the work if no accounts with that language exist
+      recommendations.concat(FollowRecommendation.safe.filtered.localized(locale).limit(100)) if AccountSummary.safe.filtered.localized(locale).exists?
+
+      # Use language-agnostic results if there are not enough language-specific ones
+      recommendations.concat(fallback_recommendations.take(100 - recommendations.size)) if recommendations.size < 100
```

Comment:
There is still the issue of duplicates taking up space from the fallback set, but I expect this to be a minor issue.

(I mean, say you have 40 language-specific recommendations, the top 60 from the generic recommendations will be taken to complete the set, but any duplicate in the 60 recommendations will be wasted, e.g. if the 40 language-specific recommendations are also in the top 60 generic recommendations, only 20 new recommendations will be effectively added)